online.hatsune_miku.tachidesk-vaadinui.json

```diff
@@ -0,0 +1,63 @@
+{
+  "app-id": "online.hatsune_miku.tachidesk-vaadinui",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "23.08",
+  "sdk": "org.freedesktop.Sdk",
+  "sdk-extensions": [
+    "org.freedesktop.Sdk.Extension.openjdk17"
+  ],
+  "command": "vaaui.sh",
+  "modules": [
+    {
+      "name": "openjdk",
+      "buildsystem": "simple",
+      "build-commands": [
+        "/usr/lib/sdk/openjdk17/install.sh"
+      ]
+    },
+    {
+      "name": "Tachidesk-VaadinUI",
+      "buildsystem": "simple",
+      "build-commands": [
+        "mkdir -p /app/bin",
+        "ls",
+        "mv Tachidesk-VaadinUI-*.*.*.jar /app/bin/Tachidesk-VaadinUI.jar",
```

Comment:
Add `dest-filename: Tachidesk-VaadinUI.jar` to the source and drop the `*` pattern